tools/analyzers/telemetryconvcheck/telemetryconvcheck.go

```diff
@@ -0,0 +1,201 @@
+package telemetryconvcheck
+
+import (
+	"flag"
+	"go/ast"
+	"go/token"
+	"go/types"
+	"strings"
+
+	"github.com/samber/lo"
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+const otelconvPackage = "github.com/authzed/spicedb/internal/telemetry/otelconv"
```

Comment:
Could we make this a configurable flag? Otherwise, if we want to re-use this analyzer in another repo, we would have to come edit SpiceDB